[2020-06-01 20:15:23,026] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: hello.get_covid_data_task 2020-06-01T14:44:42.696601+00:00 [queued]>
[2020-06-01 20:15:23,034] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: hello.get_covid_data_task 2020-06-01T14:44:42.696601+00:00 [queued]>
[2020-06-01 20:15:23,035] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-01 20:15:23,035] {taskinstance.py:880} INFO - Starting attempt 1 of 2
[2020-06-01 20:15:23,035] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-01 20:15:23,279] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): get_covid_data_task> on 2020-06-01T14:44:42.696601+00:00
[2020-06-01 20:15:23,315] {standard_task_runner.py:53} INFO - Started process 3818 to run task
[2020-06-01 20:15:23,879] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: hello.get_covid_data_task 2020-06-01T14:44:42.696601+00:00 [running]> LAPTOP-P8ICNHVB.localdomain
[2020-06-01 20:15:25,906] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-06-01 20:15:25,911] {taskinstance.py:1052} INFO - Marking task as SUCCESS.dag_id=hello, task_id=get_covid_data_task, execution_date=20200601T144442, start_date=20200601T144523, end_date=20200601T144525
[2020-06-01 20:15:32,924] {logging_mixin.py:112} INFO - [2020-06-01 20:15:32,923] {local_task_job.py:103} INFO - Task exited with return code 0
